### DISCUSSION TITLE
Grpc exception related to http2 local docker tye setup

### DISCUSSION THREAD

**Discussion by shashikantpawar21** [Eventuous/eventuous#127] [Q&A]
I have started getting this grpc call error multiple times when i restart the application.. 
Want to understand how to get rid of it and what the impact of this error 

![image](https://user-images.githubusercontent.com/83539291/186746857-7291b9da-4f26-4e35-b89c-683f86809183.png)

**Comment by JulianMay** [comment]
This is not specific to Eventous, but EventStore - Try asking at [Event Store's discussion forum](https://discuss.eventstore.com)

**Reply by shashikantpawar21** [reply]
Thank you i posted the error

**Reply by shashikantpawar21** [reply]
Just a info ..this started occuring after the upgrade of eventuous to 0.10

Should something to be done in library to take care of this as it never happened with 0.6 version  of eventuous
